### ISSUE TITLE
Make trivialEncrypt private in TFHE.sol and remove it from docs

### ISSUE THREAD

**Issue by jatZama** [zama-ai/fhevm-solidity#675]
We would like to NOT expose the trivial encryption feature anymore to devs using `TFHE.sol` library.
We need to be very careful not missing cases of handling uninitialized encrypted variables, for eg today we are missing this in `TFHE.select` at least (and maybe other ops?).

**Comment by jatZama** [comment]
there is a closely related question to this issue btw from all the contracts I reviewed from fhevm-contracts : we tried to limit as much as possible the use of trivial encrypt, but I noticed that sometimes when a user request a reencrypt or a dapp tries to read a handle from a view function to use it in computation, from a handle which is supposed to belong to him/it - for instance `balances[msg.sender]` but if it is still uninitialized  - we should take extra-care when we will remove completely the trivialEncrypt from TFHE.sol. At least this would mean that both dApp contracts calling other contract as well as front-end devs handling reencrypt request, should put extra-logic for handling uninitialized variables. Something important to keep in mind.

**Comment by yuxizama** [comment]
Hey @jatZama , is it still something onoging?

**Comment by jatZama** [comment]
I don't think we have a definitive answer on this issue. The goal was to remove exposition to trivialEncrypt for dapp developers, to reduce confusion, but @immortal-tofu did not give a definitive agreement neither refusal on this as far as I remember.
